static/templates/confirm_dialog/confirm_moving_messages.hbs

```diff
@@ -0,0 +1,19 @@
+<p>
```

Comment:
I think there's a better way to handle plurals for internationalization. https://zulip.readthedocs.io/en/latest/translating/internationalization.html#frontend-translations

Reply:
We should also split this into two separate translation blocks, to reduce duplication for translators.

Reply:
Updated the code to use `plural` format.

And I am not sure what you meant by "spllit into two separate translation blocks", do we want to tag different sentences for translation separately?

Reply:
Yeah, the suggestion is to tag the sentence with the first plural separately from the sentence with the second plural, so that each sentence can be translated once.